configs/raster/ch_swisstopo_fixpunkte-agnes.cfg

```diff
@@ -0,0 +1,25 @@
+[Database]
+dbName: kogis_int
+dbSchema: fpds
+tableName: agnes
+
+[Grid]
+gridOrigin: topLeft
+terrainBased: False
+bucketPath: 1.0.0/ch.swisstopo.fixpunkte-agnes/default/20140924/4326/
+sridFrom: 4326
+# The srid of the table
+sridTo: 21781
+pxTolerance: 10
```

Comment:
https://github.com/geoadmin/bgdi-tilegenerator/blob/master/etc/tilecache.swisstopo.cfg#L345
And it's actually 20 :)